waku/waku_test.go

```diff
@@ -1102,8 +1144,12 @@ func TestHandleP2PMessageCode(t *testing.T) {
 		case err := <-errorc:
 			t.Log(err)
 		case <-time.After(time.Second * 5):
-			rw1.Close()
-			rw2.Close()
+			if err := rw1.Close(); err != nil {
+				t.Error(err)
+			}
+			if err := rw2.Close(); err != nil {
+				t.Error(err)
+			}
```

Comment:
[Error handling is good](https://github.com/status-im/status-go/pull/1957/files#r417328424)